lrauv_ignition_plugins/src/ScienceSensorsSystem.cc

```diff
@@ -1557,84 +802,109 @@ void ScienceSensorsSystem::PostUpdate(const ignition::gazebo::UpdateInfo &_info,
     if (this->dataPtr->spatialOctrees[this->dataPtr->timeIdx]->getLeafCount()
       > 0)
```

Comment:
This test always evaluates to true, meaning the code below that allows the subsequent interpolation to happen never executes.

Reply:
Sorry that was a refactor error. Supposed to break on the opposite condition.

Reply:
Fixed in 3d7dfb5